srcpkgs/EncryptionWizard/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'EncryptionWizard'
+pkgname=EncryptionWizard
+version=3.5.2
+revision=1
+noarch=yes
+build_style=fetch
+hostmakedepends="unzip wget"
+depends="virtual?java-runtime"
+short_desc="Java-based file and folder encryption software"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="Proprietary"
```

Comment:
Not sure what on the license, since it's 1) free for public use 2) probably subject to FOIA

Reply:
Went with GOTS-Public